cartographer/mapping/proto/map_limits.proto

```diff
@@ -14,7 +14,7 @@
 
 syntax = "proto3";
 
-import "cartographer/mapping/proto/2d/cell_limits.proto";
+import "cartographer/mapping/proto/cell_limits.proto";
```

Comment:
Correct. It's the number as a first character which Python does not like combined with the fact that protobuf keeps the directory structure in Python.

Regarding `cell_limits_2d.proto`: I was trying to keep the change minimal, but I can rename it if you think this would be a better name.

The suggestion with `grid_2d` I do not understand: the `2d` subdirectory contains many different proto files, not just for the grid.

Regarding additional subdirectories for structure: separate concern, so follow up PR?

Reply:
Renamed to `cell_limits_2d.proto`.